ComIOP/Common/Client/Da/ComDaClientNodeManager.cs

```diff
@@ -825,7 +825,6 @@ protected override ServiceResult CreateMonitoredItem(ServerSystemContext context
             monitoredItem = datachangeItem;
 
             // save the monitored item.
-            MonitoredItems.Add(monitoredItemId, datachangeItem);
```

Comment:
Why did you remove the line?

Reply:
The MonitoredItems collection of the CustomNodeManager does not exist any with the new nuget Version